### ISSUE TITLE
Why can't my training loss decrease with ArcFace? It drops to about 21 and stops falling.

### ISSUE THREAD

**Issue by LHH20000923** [fdbtrs/PocketNet#3]

**Comment by LHH20000923** [comment]
I use CASIA-WebFace, optim is SGD ,lr=0.1, batchsize=256/512, and didn't use knowledge distillation。
The loss curve is shown below。Is it a problem with the data set？
![1](https://user-images.githubusercontent.com/83012544/150628702-7a362fb7-3125-45b4-b8bb-9dea73fd411d.jpg)

**Comment by fdbtrs** [comment]
For casia dataset, you may need to train the model for at least 50 epochs with lr step of 20 30 and 40. The reported FR result in the paper is based on MS1Mv2
